pyls/workspace.py

```diff
@@ -66,21 +74,21 @@ def syspath_for_path(self, path):
         Since the workspace root may not be the root of the Python project we instead
         append the closest parent directory containing a setup.py file.
         """
-        files = config.find_parents(self.root, path, ['setup.py']) or []
+        files = config.find_parents(self._root_path, path, ['setup.py']) or []
         path = [os.path.dirname(setup_py) for setup_py in files]
         path.extend(sys.path)
         return path
 
     def is_in_workspace(self, path):
-        return not self.root or os.path.commonprefix((self.root, path))
+        return not self._root_path or os.path.commonprefix((self._root_path, path))
```

Comment:
Removed this one, it used to be used by find_parents, but now that checks itself without needing a workspace.